src/BackupMetadata.cs

```diff
@@ -84,41 +83,44 @@ public class BackupMetadata : ICloneable
 
     internal BackupMetadata() { }
 
-    protected BackupMetadata(BackupMetadata backup)
+    internal BackupMetadata(BackupMetadata backup)
     {
-       CheckpointLsn = backup.CheckpointLsn;
-       DatabaseBackupLsn = backup.DatabaseBackupLsn;
-       DatabaseName = backup.DatabaseName;
-       FirstLsn = backup.FirstLsn;
-       LastLsn = backup.LastLsn;
-       PhysicalDeviceName = backup.PhysicalDeviceName;
-       ServerName = backup.ServerName;
-       StartTime = backup.StartTime;
-       BackupType = backup.BackupType;
+      CheckpointLsn = backup.CheckpointLsn;
+      DatabaseBackupLsn = backup.DatabaseBackupLsn;
+      DatabaseName = backup.DatabaseName;
+      FirstLsn = backup.FirstLsn;
+      LastLsn = backup.LastLsn;
+      ServerName = backup.ServerName;
+      StartTime = backup.StartTime;
+      BackupType = backup.BackupType;
     }
 
     // used during testing
     public object Clone()
     {
       return MemberwiseClone();
     }
+ }
+
+  public class SingleBackup : BackupMetadata
+  {
+    public string PhysicalDeviceName { get; set; }
   }
 
-  public class StripedBackupSet : BackupMetadata
+  public class StripedBackup : BackupMetadata
   {
-    public IEnumerable<BackupMetadata> StripedBackups { get; private set; }
+    public IEnumerable<SingleBackup> StripedBackups { get; }

```

Comment:
I could see this being a `List<string>` which only contains the PhysicalDeviceName's .... but ok this works fine too.
In the constructor, you would convert by doing `stripedBackups.Select(i=> i.PhysicalDeviceName);`
If it involves more refactoring, don't worry about it.  If it's simple, I think it'll simplify the meaning here